tests/test_validation_framework/test_adapters.py

```diff
@@ -264,3 +265,17 @@ def test_timezone_removal():
 
     reader_clim = AnomalyClimAdapter(tz_reader, columns=["data"])
     assert reader_clim.read_ts(0) is not None
+
+
+def test_column_comb_adapter():
+    ds = TestDataset("", n=20)
+    orig = ds.read()
+    ds_adapted = ColumnCombAdapter(ds, func=pd.DataFrame.mean, columns=["x", "y"],
+                                func_kwargs={'skipna': True}, new_name='xy_mean')
+    ds_mean1 = ds_adapted.read_ts()
+    ds_mean2 = ds_adapted.read()
+
+    for ds_mean in [ds_mean1, ds_mean2]:
+        nptest.assert_almost_equal(ds_mean["x"].values, orig["x"].values)
+        nptest.assert_almost_equal(ds_mean["y"].values, orig["y"].values)
```

Comment:
These shouldn't change at all, right? So can you use `nptest.assert_equal` instead of `nptest.assert_almost_equal`?